infrastructure/stacks/development-pipeline/data.tf

```diff
@@ -22,4 +26,9 @@ data "aws_iam_role" "pipeline_role" {
 locals {
   deploy_envs = toset(["dev", "test", "perf"])
   to_build    = toset(["event-sender", "event-processor", "fifo-dlq-handler", "orchestrator", "cr-fifo-dlq-handler", "test-db-checker-handler", "event-replay", "authoriser", "dos-api-gateway"])
+  independent_build_images = {
+    tester = {
+      "filepath" = "^*.txt$"
```

Comment:
all text files?

Reply:
I've tried multiple version such as `^*requirement*$` but could only get that to work. It's not a large issue as we barely have an txt files in our repo

Reply:
Fixed and tested, Now only runs on files changes to files with `requirement` in the filepath